### ISSUE TITLE
PageTabBarController's indicator line bounce problem

### ISSUE THREAD

**Issue by okankocyigit** [CosmicMind/Material#768]
https://drive.google.com/file/d/0B5L3zd-GHgiFYl95OVFWcXlydkE/view

When I just make three tabs it works as expected without bouncing.

**Comment by okankocyigit** [comment]
I couldn't get the point of this [line](https://github.com/CosmicMind/Material/blob/82f8768481ccea0908bca51bf49e1d111b3038ee/Sources/iOS/PageTabBarController.swift#L380),

```
let x = (scrollView.contentOffset.x - view.width) / scrollView.contentSize.width * view.width
pageTabBar.line.center.x = selected.center.x + x
```

there might be a bug, I've also tried it on Material Samples, when I increase tab count more then three it starts animating indicator wrongly.

**Comment by danieldahan** [comment]
Thank you. I will give this a look :)

**Comment by danieldahan** [comment]
This component has now been updated in [Material 2.8.0](https://github.com/CosmicMind/Material/releases/tag/2.8.0). Features like swiping have not been included in 2.8.0, and will be included in a following release. The overall TabsController adds stability and scrollable Tabs. I am closing this issue and thank you for your patience.
